### PR TITLE
NO-JIRA: create .asf.yaml

### PR DESCRIPTION
**Pull request by jiridanek**
https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+github:
+  description: "Mirror of Apache Qpid ProtonJ2"
+  homepage: https://qpid.apache.org
+  labels:
+    - qpid
+    - proton
+    - apache
+    - amqp
+    - jms
```
Comment (gemmellr) [inline .asf.yaml]: not so much?

Comment (jiridanek) [reply .asf.yaml]: yeah, right, not jms

File: .asf.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+github:
+  description: "Mirror of Apache Qpid ProtonJ2"
+  homepage: https://qpid.apache.org
```
Comment (jiridanek) [inline .asf.yaml]: protonj2 does not seem to have components page on the website

Comment (tabish121) [reply .asf.yaml]: https://qpid.apache.org/proton/index.html

Comment (jiridanek) [reply .asf.yaml]: I somehow expected to find protonj2 on the same level as qpid jms at https://qpid.apache.org/components/

**Comment by gemmellr** [comment]
can you please squash the changes

**Comment by gemmellr** [comment]
Yeah I meant without doing that :)

**Comment by jiridanek** [comment]
Eh...

**Pull request merged**